### PR TITLE
Topnav template

### PR DESCRIPTION
**Pull request by nienn**
Template for the topnav.
Additionally, moved the topnav to an include in base.html.

Note: I'm moving things from the mockups to templates adapting just the required code. So the actual topnav code is mostly unchanged.

### CODE REVIEW

**Review by agjohnson — COMMENTED** [review]
Looks good! We should update the SVGs here and in guidelines to remove the inkscape extraneous bits from my local machine. I missed those.

File: readthedocs-theme/static/img/logo-wordmark-dark.svg [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="2000"
+   height="400"
+   id="svg5241"
+   version="1.1"
+   inkscape:version="0.48.5 r10040"
+   sodipodi:docname="rtd-logo-wordmark-large-dark.svg"
+   inkscape:export-filename="/home/anthony/secure/dev/readthedocs-corporate/readthedocsinc/corporate/static/corporate/img/header-logo.png"
```
[the remark this answers is not in the document]
Comment (nienn) [reply readthedocs-theme/static/img/logo-wordmark-dark.svg:18]: True. Illustrator also likes to inject a lot of garbage. I normally clean everything.

[the remark this answers is not in the document]
Comment (nienn) [reply readthedocs-theme/static/img/logo-wordmark-dark.svg]: 👍 Done.

File: readthedocs-theme/static/img/logo-wordmark-dark.svg [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+     id="layer1"
+     transform="translate(0,-652.3622)">
+    <g
+       id="g5310"
+       transform="translate(-7.2877533,-6.3546821e-5)"
+       style="fill:#32322a;fill-opacity:1">
+      <g
+         inkscape:export-ydpi="90"
+         inkscape:export-xdpi="90"
+         inkscape:export-filename="/home/anthony/secure/docs/rtd/assets/rtd-logo-2.png"
```
[the remark this answers is not in the document]
Comment (nienn) [reply readthedocs-theme/static/img/logo-wordmark-dark.svg]: Done too.

**Review by agjohnson — APPROVED** [review]

File: readthedocs-theme/static/img/logo-wordmark-dark.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (humitos) [reply readthedocs-theme/static/img/logo-wordmark-dark.svg]: I opened an issue to do this as well in https://github.com/readthedocs/guidelines/issues/5. Someone wanted to use that file but told me "the SVG is not optimized for the web" :)

**Pull request merged**